lib/src/model/regular_polygon.dart

```diff
@@ -106,9 +107,6 @@ class RegularPolygon {
   /// See the [Corner] class for more.
   final Corner corners;
 
-  /// The angle between two sides of the polygon.
-  double get innerAngle => 360 / sides;
```

Comment:
Think about leaving this as public, since just removing a member could be a breaking change and would normally necessitate a major version incrementation. Since this is still major version 0, it would be fine to deprecate this, but think about what the `RegularPolygon` class should provide or not.